### DISCUSSION TITLE
제페토 빌드 에러

### DISCUSSION THREAD

**Discussion by Asipe42** [naverz/zepeto-studio-kor#1042] [5. Q&A]
Play on Zepeto 아이콘을 누르니 로딩 후, 아래와 같이 에러가 나옵니다

해결 방법을 알 수 있을까요?
![Error](https://user-images.githubusercontent.com/86538738/179907313-eec03545-e24d-4475-b20f-47d53873928a.PNG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 스샷으로 보았을 때는 메타파일 오류와, System.Reflection.TargetInvocationException 오류로 보여집니다.
System.Reflection.TargetInvocationException 오류는 처리하지 않은 예외가 발생했을 때 비정상적인 종료가 발생하는 것으로
처리하지 않은 예외를 다루는 이벤트를 추가함으로서 해결 될 수 있습니다.
보다 상세한 내용 파악을 위해서는 이슈 게시판에 개발 환경 및 사용하신 코드등을 첨부해주시면 도움이 될 것 같습니다.
감사합니다.
